src/utils/subscriberBreaches.ts

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getUserEmails } from "../db/tables/emailAddresses.js";
+import { HibpLikeDbBreach, getBreachesForEmail } from "./hibp.js";
+import { getSha1 } from "./fxa.js";
+import { filterBreachDataTypes } from "./breachResolution.js";
+import {
+  Breach,
+  Subscriber,
+} from "../app/(nextjs_migration)/(authenticated)/user/breaches/breaches.js";
+
+interface SubscriberBreach {
+  addedDate: string;
+  breachDate: string;
+  dataClasses: Array<string>;
+  description: string;
```

Comment:
Q: Do we use the HIBP `description` anywhere?

Reply:
Our /breach-details pages (like https://monitor.firefox.com/breach-details/linkedin) have a generic description like:
> _"On ⁨May 5, 2012⁩, ⁨LinkedIn⁩ was breached. Once the breach was discovered and verified, it was added to our database on ⁨May 21, 2016⁩."_

Whereas the `Description` from HIBP (https://haveibeenpwned.com/api/v3/breach/linkedin) is more like this (and doesn't get localized or displayed, as far as I know):
> _"In May 2016, <a href="https://www.troyhunt.com/observations-and-thoughts-on-the-linkedin-data-breach" target="_blank" rel="noopener">LinkedIn had 164 million email addresses and passwords exposed</a>. Originally hacked in 2012, the data remained out of sight until being offered for sale on a dark market site 4 years later. The passwords in the breach were stored as SHA1 hashes without salt, the vast majority of which were quickly cracked in the days following the release of the data."_

So maybe the question is whether this is the HIBP description or our description. https://github.com/mozilla/blurts-server/blob/main/src/app/(nextjs_migration)/(guest)/breach-details/%5BbreachName%5D/page.tsx
versus https://github.com/mozilla/blurts-server/blob/61d0cfa80220c7809d4f850df4f3770b4030f518/locales/en/app.ftl#L535